etrago/execute/__init__.py

```diff
@@ -418,6 +418,15 @@ def optimize(self):
 
         self.grid_optimization()
 
+    elif self.args["method"]["type"] == "sclopf":
+        self.sclopf(
+            post_lopf=False,
+            extra_functionality=None,
```

Comment:
The function `iterate_sclopf()` is failing due to an unexpected keyword. Error message:
```

  File ~/env/etrago_710_fail-fix/lib/python3.10/site-packages/spyder_kernels/py3compat.py:356 in compat_exec
    exec(code, globals, locals)

  File ~/git/eTraGo/etrago/appl.py:770
    etrago = run_etrago(args, json_path=None)

  File ~/git/eTraGo/etrago/appl.py:749 in run_etrago
    etrago.optimize()

  File ~/git/eTraGo/etrago/execute/__init__.py:422 in optimize
    self.sclopf(

TypeError: iterate_sclopf() got an unexpected keyword argument 'extra_functionality'
```